src/main.cpp

```diff
@@ -1667,10 +1667,16 @@ void vcRenderSceneUI(vcState *pProgramState, const ImVec2 &windowPos, const ImVe
       vcMain_ProfileMenu(pProgramState);
 
       if (vcMenuBarButton(pProgramState->pUITexture, vcString::Get("menuNewScene"), vcB_Invalid, vcMBBI_NewProject, vcMBBG_SameGroup))
+      {
+        pProgramState->activeTool = vcActiveTool_Select;
```

Comment:
From memory, there's a common code path when create and load project are successful, could this be put there instead? That way, any time a project changes (dropping a JSON file into the scene for example) this will always happen, instead of in the cases we've handled.

Reply:
Moved the deactivation of the Active Tool in vcProject_Deinit()